### ISSUE TITLE
Quokka hangs with mathjs

### ISSUE THREAD

**Issue by thunderkid** [wallabyjs/quokka#460]
### Issue description or question
I'm trying to run [mathjs ](https://www.npmjs.com/package/mathjs) in a quokka file. If I try to import anything at all from mathjs, then nothing else seems to execute - it just hangs. Nothing beyond the first line appears in the output window, and nothing executes in the file itself - ie nothing appears after my //? prompts.
I have tried this on two separate Windows 10 machines, with 64GB RAM. Quokka works fine with other npm modules, and mathjs is executing fine within a webpack'd application. It seems to be just the combination of Quokka + mathjs that fails.
### Sample code
```
import {complex} from 'mathjs';
2 + 3 //?              <---------------------- nothing appears from this point on
const b = complex('4 - 2i') //?
```

### Quokka.js Console Output
```
Quokka 'mathJsLab.ts' (node: v8.12.0, TypeScript: v3.7.5)​​​​
```
nothing appears after this line - it just hangs.
### Code editor version

Visual Studio Code v1.41.1
### OS name and version

Windows10 build 1809

**Comment by ArtemGovorov** [comment]
We have tried your code sample and it is working as expected:
<img width="498" alt="Screen Shot 2020-01-26 at 7 40 00 pm" src="https://user-images.githubusercontent.com/979966/73133384-aa409f80-4073-11ea-86ca-2da8d4d00d0c.png">

Could you please create a sample repo where we could reproduce the issue?

**Comment by thunderkid** [comment]
So I've finally managed to isolate the problem some more. For me it seems to occur only when both `ts-node` is specified, and when `allowJs` is set. [Here's a repo](https://github.com/thunderkid/quokkaMathjsBug/tree/master).

To reproduce, 
```npm install```
then start quokka on mathJsLab.ts. It will hang.
However, if you either remove `ts-node` from the `devDependencies` in `package.json`, OR if you set `allowJs: false` in `tsconfig.json`, then it will work.

**Comment by NikGovorov** [comment]
Thanks for providing the repo and the detailed set of steps to reproduce the problem. We were able to duplicate the problem and have released a new version of Quokka for VS Code that fixes the problem (v1.0.275).
